### ISSUE TITLE
[feature request] option to make search global and not tab specific

### ISSUE THREAD

**Issue by avatar1024** [FossifyOrg/Phone#150]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Phone/issues) or [closed](https://github.com/FossifyOrg/Phone/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Phone/discussions) or [closed](https://github.com/FossifyOrg/Phone/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Phone/blob/master/CONTRIBUTING.md).
- [X] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

Most often the dial app opens by default on the call history tab (as I often either call someone I've already called recently, or call back someone that's just called me). However if I want to call someone that's not in there I naturally use the search bar to search that contact but with the current behaviour gets an empty result as it searches only in the tab I'm in (Call history). So I have to go to the contact tab and then search again. This feels like a rather counter-intuitive behaviour, at least for my workflow.

While I'm sure some user like the current behaviour, it would be nice to have the option to make the search bar a global search across all tabs. Especially as this "global search" behaviour requested here is by the way the default behaviour of the main dial app on stock android and lineage os and so I expect this is what users would be most used to (including myself, and I was rather confused by the behaviour of the Simple Contact / Dial app in that respect)

### Why do you want this feature?

To make the experience more intuitive and faster, as well as aligned with other contact apps.

### Additional information

Request on SMT: https://github.com/SimpleMobileTools/Simple-Contacts/issues/1031

**Comment by Aga-C** [comment]
Wouldn't one of these two requested features be enough for you?
- https://github.com/FossifyOrg/Phone/issues/86
- https://github.com/FossifyOrg/Phone/issues/94

**Comment by NickAndreou** [comment]
Search should work the same in both contacts and history, right now it is inconsistent and confusing. Just make search look for contacts as it works in contacts and it would be fine. If the point is to find the last time you have called someone that may be an extra line in the contact display and that's it.

**Comment by ksz16** [comment]
The lack of a global search option in each tab is currently the Phone app's biggest problem, which makes it very difficult to use it for me. Additionally, it would be nice if clicking on a search result in the Favorites and Call History tabs would make a direct call.

**Comment by Cwpute** [comment]
This would tremendously improve my use of this app !
Also something to consider: it doesn't help that both Contacts and Phone apps use almost the same layout can confuse me further as i can asrume i'm on the phone app when i'm using the contacts app and vice-versa… Maybe something can be dome in that regard too.

**Comment by Limfjorden** [comment]
I agree with everyone here. It also seems that if I add a nickname to a contact who is in my call log, I cannot find them in Fossify Phone's search field by that nickname. This is not the case when adding a phone number to a contact, as then i can find previous calls by searching the name. This inconsistency should imo be remedied.

**Comment by carlnickel** [comment]
I would love this feature too. Do we have anybody to work on this?

**Comment by wooque** [comment]
Yes, multiple times a did search on favorites tab, got no results and thought I don't have a number, but then few moments later I realized search on favorites only searches favorites and not all numbers, so I have to switch tabs and do search again. 
Extremely unintuitive.

**Comment by NickAndreou** [comment]
Wooque, it seems this cross-tabs search feature has been implemented as I see it working in PHONE.
